Simulator/kinematics.py

```diff
@@ -34,6 +34,7 @@ def calc_w_dot(w, torque, I):
         w_dot: derivative of angular velocity (rad/s/s) np.array[3x1][float64]
     """
 
-    w_dot = np.linalg.solve(I, -np.dot(skew(w), np.dot(I, w)) + torque)
+    # w_dot = np.linalg.solve(I, -np.dot(skew(w), np.dot(I, w)) + torque)
+    w_dot = np.linalg.inv(I) @ -(skew(w) @ (I @ w)) + torque
     return w_dot
```

Comment:
Also, in the future should we be using @ operator instead of dot product here? If I understand correctly, these are identical operations for vector-matrix multiplication

Reply:
They are identical. This is more for readability so there aren’t dots of dots which are sometimes hard to decipher.